policyengine_us/variables/gov/states/nj/tax/income/credits/eitc/nj_childless_eitc_age_eligible.py

```diff
@@ -27,11 +23,24 @@ def formula(tax_unit, period, parameters):
         age = person("age", period)
         min_age = p.eligibility.age.min
         max_age = p.eligibility.age.max
-        meets_age_requirements = (age >= min_age) & (age <= max_age)
+        age_eligible = (age >= min_age) & (age <= max_age)
 
-        return (
-            ~separate
-            & no_qualifying_children
-            & tax_unit.any(meets_age_requirements)
-            & tax_unit("nj_eitc_income_eligible", period)
+        # federal EITC conditions without age and a qualifying child
+        eitc = parameters.gov.irs.credits.eitc(period)
+        investment_income_eligible = tax_unit(
```

Comment:
we also might need to do this in a separate variable as this is just for the age eligibility